test/unit/instrumentation/http/no-parallel/queue-time.test.js

```diff
@@ -22,7 +22,12 @@ tap.test('built-in http queueTime', (t) => {
   let THRESHOLD = null
 
   t.beforeEach((done) => {
-    agent = helper.instrumentMockedAgent()
+    // implicitly disabling distributed tracing to match original config base settings
+    agent = helper.instrumentMockedAgent({
+      distributed_tracing: {
+        enabled: false
+      }
+    })
```

Comment:
I'm kinda surprised this broke something here... unless it is testing something Cross Application Tracing related in here? Although, I'm not sure what that has to do with queue time?

Curious to learn more here.

Assuming there's a valid reason and not just an unnecessary assertion in there... minor, but I don't think you need that comment about 'original config base settings.' It won't mean anything to anyone in the future as the significance of actually why it was disabled is not clear.

Reply:
I'll go ahead and take a quick pass and rework the tests you mentioned above and remove the comments on the test setup.

Reply:
You're right, enabling DT did not affect this test. Must have updated on accident.